docs/references/making_release.md

```diff
@@ -1,16 +1,16 @@
 ## How to make a new release on PyPI and conda-forge
 
 ### PyPI
-1. iterate the version number. Our version numbering approach is:
+1. update the changelog to put all the unreleased changes under the new version (leaving the unreleased section empty).
```

Comment:
This step requires that you've decided on a next version number, but the discussion about how to do that is in step 3...

Reply:
I will move the information about defining the next release to the top.